packages/oslo-converter-uml-ea/lib/utils/utils.ts

```diff
@@ -8,7 +9,11 @@ export function ignore(object: EaObject): boolean {
   return Boolean(ignoreObject);
 }
 
-export function getTagValue(object: any, tagName: TagNames, _default: any): string {
+export function getTagValue(
+  object: any,
```

Comment:
Maybe nitpicky, but these `any` types aren't interesting. Is it because the value can be a wide range of values that you did it? If so, then it's understandable, just feels off 🛩️